canonical/juju.is
# Operator day

## Done

- Updated Operator day page

## QA

- Visit the demo pages:
  - https://juju-is-450.demos.haus/operator-day
  - https://juju-is-450.demos.haus/operator-day/kubecon-na-2022
  - https://juju-is-450.demos.haus/operator-day/kubecon-eu-2022
  - https://juju-is-450.demos.haus/operator-day/kubecon-na-2021
  - https://juju-is-450.demos.haus/operator-day/kubecon-eu-2021
  - https://juju-is-450.demos.haus/operator-day/kubecon-na-2020
- ensure everything is accurate according to the [copy doc](https://docs.google.com/document/d/1SVZS3FmGqWmgZXqnUTXgQScGFXDdtjonfSAAAMUEYlk/edit#)


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/CC-491
Fixes https://warthogs.atlassian.net/browse/CC-492
Fixes https://warthogs.atlassian.net/browse/WD-2020
